src/test/tests_common.h

```diff
@@ -45,9 +45,31 @@
 
 #include "crt_internal.h"
 
+#define DBG_PRINT(x...)                                                 \
+	do {                                                            \
+		if (opts.is_server)                                     \
+			fprintf(stderr, "SRV [rank=%d pid=%d]\t",       \
+			opts.self_rank,                                 \
+			opts.mypid);                                    \
+		else                                                    \
+			fprintf(stderr, "CLI [rank=%d pid=%d]\t",       \
+			opts.self_rank,                                 \
```

Comment:
clients in non-pmix mode do not really have a rank, so printing one is not helpful - normally it will be 0

Reply:
Right, but I think it is nice to have the srv and cli prints alligned. It's just easier on the eyes.